Solutions/Corvus.Json.ExtendedTypes/Corvus.Json/Internal/JsonConverter{T}.cs

```diff
@@ -0,0 +1,39 @@
+﻿// <copyright file="JsonConverter{T}.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+using System.Text.Json;
+
+namespace Corvus.Json.Internal;
+
+/// <summary>
+/// Generic json converter for <see cref="IJsonValue{T}"/>.
+/// </summary>
+/// <typeparam name="T">The type of <see cref="IJsonValue{T}"/> to convert.</typeparam>
+public class JsonConverter<T> : System.Text.Json.Serialization.JsonConverter<T>
```

Comment:
I'm wondering if this should be called `JsonValueConverter`. And not just because I don't like it when we defined types with names that clash with .NET runtime class library names (particularly when the corresponding clash is with a namespace clients are highly likely to have in scope). This is specifically a converter for `IJsonValue<T>`, and it would be good for it (and the associated configuration settings class, `JsonConverter`) to reflect that.

Reply:
I agree and have renamed in the conversion.